test/e2e/apps/disruption.go

```diff
@@ -164,21 +164,7 @@ var _ = SIGDescribe("DisruptionController", func() {
 
 			// Locate a running pod.
 			var pod v1.Pod
```

Comment:
You can remove this line now.

Reply:
Thanks, was useful whilst developing the tests but sure can be removed now.